tests/PilotLive/Models/BaseModelTest.php

```diff
@@ -114,15 +96,12 @@ protected function verifySetReadOnly(string $class, string $field)
     protected function verifyGetUndefined(string $class)
     {
         $className = $this->getClassName($class);
-
```

Comment:
Yep, it's important to use newlines to group logic chunks of functionality in a method and make for lower cognitive load on developers when working on the code in the future.

Always remember that we write code for humans 😎